.travis.yml

```diff
@@ -0,0 +1,18 @@
+sudo: true
+language: java
+jdk:
+  - oraclejdk8
+before_install:
+ - sudo mkdir -p /usr/local/nn-analytics/db/
+ - sudo chmod -R 777 /usr/local/nn-analytics
+matrix:
+  fast_finish: true
+  include:
+    - env: VERSION='-PhadoopVersion=2.4.0'
+    - env: VERSION='-PhadoopVersion=2.6.1'
+    - env: VERSION='-PhadoopVersion=2.7.0'
+    - env: VERSION='-PhadoopVersion=2.8.0'
+    - env: VERSION='-PhadoopVersion=2.9.0'
+    - env: VERSION='-PhadoopVersion=3.0.0'
+script:
+  - ./gradlew clean build licenseMain licenseTest findbugsMain findbugsTest buildRpm $VERSION
```

Comment:
should we add checkStyle here ?

Reply:
Yes... let’s definitely add that

Reply:
We will as a part of wrapping up #23

Reply:
Oh sorry -- it's actually already there anyway guys. Take a look at the job log here: https://travis-ci.com/paypal/NNAnalytics/jobs/127776725

You can see its already running checkstyle as part of the `build` operation.
I confirmed this locally as well. I forgot it does that.

Reply:
confirming that spotlessJava runs as part of `gradle build`, and also fails if there is violatioin. although it also runs the checkstyle for me, probably should remove it from `gradle apply plugin`.
I think we should merge this, and take care of larger checkstyle issues on issue #21 .
+1, LGTM

Reply:
Okay we are in agreement on proceeding AFAICS. I will commit shortly then. Thank you!